paf-mvp-benchmark/K6/util.js

```diff
@@ -0,0 +1,183 @@
+import http from 'k6/http';
+import { check, fail } from 'k6';
+//Cookies
+export function logCookie(cookie) {
+    // For full list of attributes see: https://k6.io/docs/using-k6/cookies#properties-of-a-response-cookie-object
+    console.log(
+        `\t${cookie.name}: ${cookie.value}\n\tdomain: ${cookie.domain}\n\tpath: ${cookie.path}\n\texpires: ${cookie.expires}\n\thttpOnly: ${cookie.http_only}`
+    );
+}
+
+/*function extractCookiesFromResponse(response){
+    let cookies = [];
+    for (const name in response.cookies) {
+        if (response.cookies.hasOwnProperty(name) !== undefined) {
+            let definition = response.cookies[name][0];
+            let cookie = {
+                name:definition.name,
+                value:definition.value,
+                options: {
+                    domain: definition.domain,
+                    path: definition.path,
+                    expires: definition.expires,
+                    secure: definition.secure
+                }
+            };
+            cookies.push(cookie);
+        }
+    }
+    return cookies;
+}*/
+
+export function logCookiesFromResponse(response) {
+    //Cookies seen by K6
+    console.log(`>>>Cookies for ${response.url}`);
+    for (const name in response.cookies) {
+        if (response.cookies.hasOwnProperty(name) !== undefined) {
```

Comment:
I think this is not correct: `hasOwnProperty` never returns `undefined`. It will return `false` if the object doesn't have the property (not `undefined`).

Reply:
Done